packages/core/src/lib/Log.ts

```diff
@@ -14,10 +14,45 @@
  *  limitations under the License.
  */
 
+/**
+ * An interface for loggers.
+ */
+export interface Log {
+
+  verbose: boolean;
```

Comment:
Why does this need to be on the interface? (also, nit: empty line above).

Reply:
That's what we talked about didn't we?
We make Log an interface and create 2 classes that will implement it: ConsoleLog(which will be exactly as log was), and MockLog

Reply:
by line above do you mean to remove the empty line on 21 or add a new one before 20?